pagefaultgames/pokerogue
# [BUG] potentially fix PokePT_Wansung font lower f

…above it)





## What are the changes?


I've fixed the PokePT_Wansung font file to remove the abomination pixels above the lowercase `f`

## Why am I doing these changes?




Lots of users have been experiencing black boxes on their screen. This might be the cause.
<img width=400 src="https://media.discordapp.net/attachments/1176874654015684739/1250588246468198420/image.png?ex=666b7c90&is=666a2b10&hm=35a1c0c8e0d1cec67068579a71cb4cf1e1df6baeacd220a91e1738367cec7fa7&=&format=webp&quality=lossless&width=2252&height=1116" />

## What did change?



I've used an online font editor to remove those pixels from the `f`
https://www.tophix.com/font-tools/font-editor

### Screenshots/Videos



#### Before

<img width=300 src="https://media.discordapp.net/attachments/1176874654015684739/1250591762519556207/image.png?ex=666b7fd7&is=666a2e57&hm=c8d2c5f5118851e8d41464b20dbb8d82816f8552a2afb89618b6e428add8f56e&=&format=webp&quality=lossless&width=2268&height=1080"/>

#### After

<img width=300 src="https://media.discordapp.net/attachments/1176874654015684739/1250591763077402654/image.png?ex=666b7fd7&is=666a2e57&hm=0869ffd5d0d564b832009aaa871cad60705e7b9e510956405eda59d2d9bb4dd1&=&format=webp&quality=lossless&width=2268&height=890"/>

## How to test the changes?





Play the game and pray that no more boxes appear.
Alternatively I'm using the [Font-Preview](https://open-vsx.org/extension/ctcuff/font-preview) vs-code extension to view the content of a font file

## Checklist
